Merge Latest PB Development Changes

manjotsidhu opened the pull request.

AndroiableDroid committed:
pb: use new thing

Signed-off-by: Mohd Faraz <androiabledroid@gmail.com>

manjotsidhu committed:
deploy: SF UNIX name rename from pitchblack-twrp to pbrp.

Signed-off-by: Manjot Sidhu <manjot.techie@gmail.com>

rokibhasansagar committed:
update-binary: Make more compatible with POSIX sh

  - Done a little bit of Beautification to installation Message
  - Removed the semicolons on line end as they are just useless in multiline commands
  - Properly indented texts with 4-character-tab
  - Replaced bash [[]] with POSIX []
  - Added Double-quote in several places to prevent globbing and word-splitting
  - Added '/dev/block/bootdevice/by-name/' as another partition finding path
  - Other POSIX sh fixes, retained a few

Signed-off-by: rokibhasansagar <rokibhasansagar2014@outlook.com>
Change-Id: I6dcaf32e92b28af3c3a1086e5591b00eb383119d

rokibhasansagar commented on updater/update-binary:
```diff
@@ -231,9 +232,11 @@ ui_print "|---------------------------------------------|";
 ui_print "|---------------------------------------------|";
 ui_print "|                  Thank You                  |";
-ui_print "|   Reboot to PitchBlack Recovery Manually    |";
+if [[ ! -f "/sbin/keycheck" ]]; then
```
Kindly edit and use single [ ] , tab as indent (no spaces), upon newly formatted update-binary.

rokibhasansagar committed:
update-binary: adapt new format changes

rokibhasansagar commented:
@manjotsidhu 
Kindly check

manjotsidhu commented:
> @manjotsidhu
> Kindly check

Have we tested the `update-binary` ?

manjotsidhu committed:
Merge branch 'pb' into pb-dev

manjotsidhu commented:
@rokibhasansagar Please get these changes tested and then we are good to merge

rokibhasansagar committed:
update-binary: Fix keycheck execution

  - As keycheck execution is mandatory now, it will use either from zip file or /sbin

rokibhasansagar reviewed (approved):
Merge-able

rokibhasansagar committed:
update-binary: Simplify keychecker function
